web/test/karma.conf.js

```diff
@@ -60,9 +60,12 @@ module.exports = function(config) {
       'bower_components/Leaflet.utfgrid/dist/leaflet.utfgrid.js',
       'bower_components/d3/d3.js',
       'bower_components/d3-tip/index.js',
-      'bower_components/world-calendars/jquery.calendars.js',
```

Comment:
I'm not sure about including this; I think this may be contributing to the broken tests, which seem to have to do with calendar dates.